### ISSUE TITLE
Fix Request: false positive in MDCPutWithoutRemove

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#235]
Example:
   ```
    try {
            Optional.ofNullable(request.getHeader(REQ_ID)).ifPresent(id -> MDC.put(REQ_ID, id)); // false positive
           // do stuff
        } finally {
            Optional.ofNullable(MDC.get(REQ_ID)).ifPresent(id -> MDC.remove(REQ_ID));
        }
```

**Comment by jborgers** [comment]
example doesn't lead to violation, only when put is done with lowercase() and remove is not. So rule is working correctly.

**Comment by jborgers** [comment]
nothing to fix, close.
